packages/components/app/styles/components/app-frame.scss

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// APP-FRAME
+//
+
+.hds-app-frame {
+  display: grid;
+  grid-template-areas:
+    "header header"
+    "sidebar main"
+    "sidebar footer";
+  grid-template-rows: auto 1fr auto;
+  grid-template-columns: auto 1fr;
+  min-height: 100vh;
+}
+
+
+// FRAME'S CONTAINERS
+
+.hds-app-frame__header {
+  z-index: 7;
```

Comment:
Curious, why the use of z-index? 

One of the things we want to make sure does not happen is that interactive elements could be accidentally hidden behind "sticky" headers or footers. 

Some reading: 

- WCAG Success Criteria: [Focus not obscured](https://www.w3.org/WAI/WCAG22/Understanding/focus-not-obscured-minimum.html)
- CSS Technique 43: [using CSS margin and scroll-margin to un-obscure content](https://www.w3.org/WAI/WCAG22/Techniques/css/C43)

Reply:
We're reproducing (with different values) what already happens in production. We need to define a hierarchy of z-index for the top-level content (they will be converted to design tokens in a follow-up task)

Reply:
With the current design/implementation, these `z-index` values have no effect as all containers have a static position. If one of them changes however (via an override, for example, or a script), having these `z-index` values can be a good fallback to make sure they overlap as expected.

Reply:
In any case, there will be a follow-up PR for handling the `z-index` values in a more systematic way (exposing them as design tokens)